.generator/templates/libraries/jersey2/ApiClient.mustache

```diff
@@ -1107,7 +1108,8 @@ public class ApiClient{{#jsr310}} extends JavaTimeFormatter{{/jsr310}} {
     } else {
       targetURL = this.basePath + path;
     }
-    WebTarget target = httpClient.target(targetURL);
+    try {
+      WebTarget target = httpClient.target(new URI(targetURL));
```

Comment:
Do we need to catch the `URISyntaxException` for all calls on `target`?
```suggestion
    try {
      WebTarget target = httpClient.target(new URI(targetURL));
    } catch (URISyntaxException e) {
      throw new ApiException(e);
    }
```

Reply:
I just split the catch and other part of the function